modelci/hub/model_loader.py

```diff
@@ -15,10 +15,16 @@
 
 import tensorflow as tf
 import torch
-
+import joblib
 from modelci.hub.utils import parse_path_plain
 
 
+def sklearn_loader(model_weight_path: Path):
```

Comment:
Maybe it's better to rename this function to `joblib_loader`? Did these models save by sklearn API?

Reply:
Yes,  we don't have a explicit model serialization standard. So I am thinking about assigning `joblib` as the only accepted way for ml frameworks. Because we use `sklearn api` for all the 3 convert functions, I named the loader as `sklearn_loader`.

Reply:
`joblib_loader` is also a good name though I doubt that it had more functional meaning than logical.